indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -566,6 +567,23 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_avg_per_question_per_track(self):
+        ratings = {}
+        abstract_ratings = (AbstractReviewRating.query
+                            .filter(AbstractReview.abstract_id == self.id,
+                                    AbstractReviewRating.review_id == AbstractReview.id)
```

Comment:
Actually, use this query. That way you also avoid spamming extra queries for each question/review:

```python
abstract_ratings = (AbstractReviewRating.query
                    .join(AbstractReviewRating.review)
                    .join(AbstractReviewRating.question)
                    .options(contains_eager('review'), contains_eager('question'))
                    .filter(AbstractReview.abstract == self, 
                            ~AbstractReviewQuestion.is_deleted)
                    .all())
```